File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Python/program1.py

num = input("Enter the number : ")

flag = 0

for itr in range(2,(num/2)+1):
    if num%itr ==0:
        flag = 1
        break


if flag == 0:
    print("The number is Prime")
else:
    print("The number is not Prime")

    

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Java/Program1.java
import java.io.*;


class Program1{
	
	public static void main(String ... args)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter the number");
		int num = Integer.parseInt(br.readLine());


		int flag =0;
		for(int itr =2 ; itr<=num/2 ;itr++){
		
			if(num%itr == 0){
				flag = 1;
				break;
			}	

		}	

		if(flag==0){
			System.out.printf("The number is Prime");
		}
		else{
			System.out.printf("The number is Not Prime");
		}
	}	

}	
